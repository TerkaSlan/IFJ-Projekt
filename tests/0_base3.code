/*Test on BOOLOP extension - double in hexa*/

class Main {

	static void run() {
		double d = 255.255e-0;
		ifj16.print(d + "\n");
	}
}