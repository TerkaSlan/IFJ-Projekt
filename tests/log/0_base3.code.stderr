ERROR: Lexical - Line: 6 - Scanner encountered an error.
